python/cudf/tests/test_csv.py

```diff
@@ -280,30 +281,14 @@ def test_csv_reader_strings_quotechars(tmpdir):
     assert(df['text'][3] == 'f,,!.,')
 
 
-def test_csv_reader_auto_column_detection(tmpdir):
```

Comment:
This is removed as I think it's now covered by other tests, such as `test_csv_reader_dtype_inference` and the `test_csv_reader_datetime`.